lib/Views/school_login.dart

```diff
@@ -1,11 +1,12 @@
 import 'dart:async';
-import 'package:flutter/material.dart';
+import 'package:flutter/material.dart' hide ModalBottomSheetRoute;
```

Comment:
Why are you naming the import?

Reply:
This import has a conflict with the Modal Bottom Sheet library and I had to use hide with that import to work. I found this solution in Stack Overflow.
Here is the error before that change:
Error (Xcode): ../../../.pub-cache/hosted/pub.dev/modal_bottom_sheet-2.1.2/lib/src/material_with_modal_page_route.dart:4:1: Error:
'ModalBottomSheetRoute' is imported from both 'package:flutter/src/material/bottom_sheet.dart' and
'package:modal_bottom_sheet/src/bottom_sheet_route.dart'.

Reply:
Actually, I just reviewed this again, I imported a new version of the library that resolves this conflict. We can actually discard that change.

Reply:
Ok, just commit the new version, and then looks good to merge